### PR TITLE
Add unit tests for RDataFrame constructors

### PR DESCRIPTION
**Pull request by shravan97**
Features in this PR : 
* Add test cases to check working of `RDataFrame` constructors

### CODE REVIEW

**Comment by etejedor** [comment]
Looks good to me, @diocas @JavierCVilla  can you have a look?

**Review by etejedor — APPROVED** [review]

**Comment by diocas** [comment]
Seems good to me.

**Comment by diocas** [comment]
Sorry

**Comment by JavierCVilla** [comment]
Should we add a test for [this case](https://github.com/shravan97/PyRDF/blob/master/PyRDF/RDataFrame.py#L18) ? Multiple files and multiple branches?

File: tests/test_rdataframe.py [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+        reqd_vec = ROOT.std.vector('string')()
+        for elem in rdf_branches:
+            reqd_vec.push_back(elem)
+
+        RDF_2 = RDataFrame("treename", "file.root", reqd_vec)
+
+        self.assertListEqual(RDF_1.args, ["treename", "file.root", reqd_vec])
+        self.assertListEqual(RDF_2.args, ["treename", "file.root", reqd_vec])
+
+    def test_incorrect_args(self):
```
Comment (JavierCVilla) [inline tests/test_rdataframe.py]: @etejedor @diocas Current RDataFrame also raises a `TypeError` when calling `RDataFrame()` with no arguments. Should it be also considered here?

Comment (etejedor) [reply tests/test_rdataframe.py]: Sure

File: tests/test_rdataframe.py [hunk 2]
```diff
@@ -0,0 +1,47 @@
+import unittest, ROOT
+from PyRDF import RDataFrame, RDataFrameException
+
+class RDataFrameConstructorTests(unittest.TestCase):
+
+    def test_integer_arg(self):
```
Comment (etejedor) [inline tests/test_rdataframe.py]: Maybe you could add another commit to this PR with the corresponding docs for this test file?

File: tests/test_rdataframe.py (hunk 1 above)
Comment (shravan97) [reply tests/test_rdataframe.py]: Sounds good @etejedor and @JavierCVilla . I'll add more test cases :+1:

File: tests/test_rdataframe.py (hunk 2 above)
Comment (shravan97) [reply tests/test_rdataframe.py]: Sure @etejedor

**Comment by shravan97** [comment]
@etejedor and @JavierCVilla , I've added a few more cases now. Could you review this ?

**Comment by JavierCVilla** [comment]
@shravan97 Looks good to me. However, don't be afraid of making several commits to the same PR, it is easier for us to review new changes and much clearer to see the progress (besides easier to revert :upside_down_face: )

Also, I think last @etejedor 's [comment](https://github.com/shravan97/PyRDF/pull/25#discussion_r196140388) is still pending:  adding corresponding docs for this test file in a different commit.

**Comment by shravan97** [comment]
Oh yes @JavierCVilla . It's coming up 👍

**Comment by shravan97** [comment]
@etejedor and @JavierCVilla . Done now 👍

**Review by etejedor — APPROVED** [review]

**Pull request merged**